### PR TITLE
add azure release 19.0.0

### PR DESCRIPTION
**Pull request by whites11**

### CODE REVIEW

**Comment by whites11** [comment]
cncf tests green

**Comment by whites11** [comment]
upgrade test passed once. Will now run all 3 e2e tests again.
I consider release components frozen now

File: azure/v19.0.0/announcement.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**Workload cluster release v19.0.0 for Azure is available**. This is the first Azure release featuring Kubernetes 1.24. For increased performance the kubernetes api-server's cpu request are changed to be the half of the **available** CPUs in the VM. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/azure-v19.0.0/).
```
Comment (whites11) [inline azure/v19.0.0/announcement.md:1]: Worth mentioning node local DNS cache maybe?

**Review by QuentinBisson — APPROVED** [review]

**Review by pipo02mix — COMMENTED** [review]
general description LGTM, did not check metadata files

**Review by pipo02mix — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**